oaknational/Oak-Web-Application
# Feat/690 smaller images on blog

## Description

- Images are smaller on homepage on mobile
- Also, remove images from the blog listing page in all views. Blog images are only shown on the homepage 

## Issue(s)

Fixes #690 

## How to test

1. Go to https://deploy-preview-820--oak-web-application.netlify.thenational.academy/ on a mobile phone
2. See blog images smaller
3. navigate to `/blog` 
4. See no images on blog list

## Screenshots

<img width="389" alt="Screenshot 2022-09-29 at 16 12 15" src="https://user-images.githubusercontent.com/2959739/193070219-56395322-45ee-4b1d-86f5-07bcb0bb0389.png">

<img width="1440" alt="Screenshot 2022-09-29 at 16 12 39" src="https://user-images.githubusercontent.com/2959739/193070243-b3790956-14c0-4ad6-be80-bf0ff80237b2.png">

## Checklist
